test/prefer-array-some.mjs

```diff
@@ -72,20 +77,14 @@ test({
 			suggestionOutput: 'if (jQuery.some(".outer > div")) {}',
 		}),
 		// Actual messages
-		{
+		invalidCase({
 			code: 'if (foo.find(fn)) {}',
-			errors: [
-				{
-					message: 'Prefer `.some(…)` over `.find(…)`.',
-					suggestions: [
-						{
-							desc: 'Replace `.find(…)` with `.some(…)`.',
-							output: 'if (foo.some(fn)) {}',
-						},
-					],
-				},
-			],
-		},
```

Comment:
This test mean to test the actual message, please revert.

Reply:
I know, I made a refactoring to use `invalidCase` function instead

Reply:
Message not asserted in that function, only messageId.

Reply:
asserted on line 82

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/635 This is why we need those tests.